e2e/common/config/kamelet_config_test.go

```diff
@@ -89,6 +90,25 @@ func TestKameletImplicitConfig(t *testing.T) {
 			Expect(DeleteSecret(ns, secretName)).To(Succeed())
 		})
 
+		t.Run("run test default config using mounted configmap", func(t *testing.T) {
+			name := "my-own-timer-source-default-config-mounted-configmaps"
+			cmName := "my-mounted-default-secret"
+
+			var cmData = make(map[string]string)
+			cmData["camel.kamelet.my-own-timer-source.message"] = "very top mounted configmap message"
+			Expect(CreatePlainTextConfigmap(ns, cmName, cmData)).To(Succeed())
+
+			Expect(KamelRunWithID(operatorID, ns, "files/TimerKameletIntegrationConfiguration.java",
+				"-p", "camel.kamelet.my-own-timer-source.message='Default message'",
```

Comment:
Why passing this property? it's the same you later provide in the configmap. Unless we want to explicitly test the precedence of the trait over the property, I don't see the reason to have it specified. Do I miss something?

Reply:
It is probably an error. Adding some precedence test is still a good idea so I could add some, WDYT ?

Reply:
Okey, that was my confusion. No I think that for the sake of this PR we should remove any ambiguity. I'd say we already cover the precedence check in some other test.